EMSBackendBridgeBundle/DependencyInjection/EMSBackendBridgeExtension.php

```diff
@@ -25,63 +25,62 @@ class EMSBackendBridgeExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
-        $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+        $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__ . '/../Resources/config'));
```

Comment:
You should use the DIRECTORY_SEPARATOR constant instead of /

Reply:
Following the Symfony coding standards (if that is actually a thing) we should not in this case.
I found a discussion related to this topic: https://groups.google.com/forum/#!topic/symfony-devs/qxZ3Rq3N030

Looking into the Symfony code, they use the exact same lines to load the service xml. However, I think this only works because we use a relative path (see implementation of FileLocator). 

The Symfony Docs also suggest the use of / in the resource path: http://symfony.com/doc/current/bundles/extension.html#using-the-load-method